### ISSUE TITLE
2 controllers not working

### ISSUE THREAD

**Issue by renb0** [nbarkhina/N64Wasm#23]
**OS:** MacOS Ventura
**Browser:** Chrome
**Controllers (Both connected via Bluetooth):**
 - 8bitdo SF30 (Seen as an Xbox Controller)
 - Dualshock 4

When I go to remap and test a button only one of the controllers works.
With the setup above, when both are connected only the Dualshock 4's input works in the remap keys, my 8bitdo controller is ignored, then when I disconnect the DS4 the 8bitdo's button presses then starts to work in the remap dialog.

Further to this, I assume there are supposed to be 2 tabs in the remap section, I only see one - despite the controllers being connected and tested in the browser on [gamepad-tester.com](https://gamepad-tester.com/) which I used to rule out OS related issues.

Screenshots of devices:
<img width="957" alt="image" src="https://user-images.githubusercontent.com/2660345/222313001-60d52089-008e-436b-8a9f-0343f397c2af.png">
<img width="960" alt="image" src="https://user-images.githubusercontent.com/2660345/222313047-35e0f012-1f05-4ea4-9000-781ff399b77f.png">

**Comment by nbarkhina** [comment]
Yes unfortunately at the moment the multiple controllers functionality assumes all the controllers are of the same type. So for example if you had 2 DualShock controllers that would work and they would both share the same mappings. I have not yet built out the ability to have different mappings for different controllers however that is something I am planning to do at some point.

**Comment by DaBiggestBozo** [comment]
I have been trying to connect a set of joy cons to it. They work fine with other programs, and seem to look well with this, but I cannot remap the c-stick or up/down/left/right.
